spec/features/user_past_order_view_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'user past order views', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.